controllers/ocmsearch_controller.go

```diff
@@ -76,6 +76,32 @@ func (r *OCMSearchReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		log.Error(err, "Postgres Deployment  setup failed")
 		return *result, err
 	}
+	log.Info("creationg for indexer svc")
```

Comment:
```suggestion
	log.Info("creating indexer svc")
```

Reply:
Removed that line - put for my debug . @SherinV